3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,87 @@
+
+//Age-ify (A future age calculator)
+
+
+console.log("A future age calculator")
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+const age = yearFuture - yearOfBirth;
+console.log(`You will be ${age} years old in ${yearFuture}`);
+
+
+
+
+
+//Goodboy-Oldboy (A dog age calculator)
+
+console.log("age of dog");
+
+const dogYearOfBirth = 2009;
+const dogYearFuture = 2018;
+const humanYears = dogYearFuture - dogYearOfBirth;
+const dogYears = humanYears * 7;
+const shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears) {
+console.log(`Your dog will be ${dogYears} dog years old in ${dogYearFuture}`);} 
+{console.log(`Your dog will be ${humanYears} human years old in ${dogYearFuture}`);}
```

Comment:
The second console log should be put in an else statement (if ). Right now it will show the two console logs no matter the value shouldShowResultInDogYears. You could also consider the indentation in the IF statement for more readability.  https://courses.cs.washington.edu/courses/cse154/18au/resources/styleguide/js/spacing-indentation-js.html

And again good job using the string literals.